framework/assets/yii.js

```diff
@@ -144,8 +144,8 @@ yii = (function ($) {
          * @param $e the jQuery representation of the element
          */
         handleAction: function ($e, event) {
-            var method = $e.data('method'),
-                $form = $e.closest('form'),
+            var $form = $e.attr('data-form') ? $('#' + $e.attr('data-form')) : $e.closest('form'),
```

Comment:
Those are actually not the same.
$e.data('form') will be "cached" in javascript; so if I later change the dom the change will not be picked up by jQuery.